frontend/src/metabase/query_builder/components/expressions/CombineColumns/ColumnAndSeparatorRow/ColumnAndSeparatorRow.module.css

```diff
@@ -0,0 +1,27 @@
+.separator {
+  flex: 0 0 auto;
+
+  input {
+    font-family: Monaco, monospace;
+  }
+}
+
+.column {
+  flex: 1;
+}
+
+.remove {
+  flex: 0 0 auto;
+  border-color: transparent !important;
```

Comment:
we agreed not to use important, but either to introduce a new variant or fix theme